addon/components/pix-checkbox.hbs

```diff
@@ -5,10 +5,11 @@
   >
     {{this.label}}
   </label>
-  <input
+  <Input
     id={{this.id}}
-    type="checkbox"
+    @type="checkbox"
```

Comment:
Je ne comprend pas pourquoi on utilise l' `@`pour `type`

Reply:
On utilise le composant <Input> fait pas ember comme mentionné dans la description de PR, celui-ci a besoin de cet argument : https://guides.emberjs.com/release/components/built-in-components/